tests/base_new_frmwrk.py

```diff
@@ -150,6 +150,217 @@ def set_replication_methods(self, conn_id, catalogs, replication_methods):
             connections.set_non_discoverable_metadata(
                 conn_id, catalog, menagerie.get_annotated_schema(conn_id, catalog['stream_id']), replication_md)
 
+    ### Method to return the fields that are non-upsert to be excluded from the all fields test as these non-upsert fields are not replicated
```

Comment:
The comment sounds like the opposite of the method.  Aren't we returning the upsert fields and excluding the non-upsert fields since they aren't replicated?

Reply:
Changed the comment as it was not very clear